packages/codemods/nuxt/4/template-compilation-changes/.codemodrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://codemod-utils.s3.us-west-1.amazonaws.com/configuration_schema.json",
+  "version": "1.0.0",
+  "private": false,
+  "name": "nuxt/4/shallow-function-reactivity",
```

Comment:
the name is incorrect... i think you copy pasted this from another codemod...

Reply:
😅 yeah all the files except the src dir was copied from the `shallow-function-reactivity` codemod, I'll fix this.